src/App.vue

```diff
@@ -59,16 +59,16 @@ export default {
   data: () => ({
     componentLinks: [
       {
-        to: '/ImageContainer',
-        label: 'Image Container',
+        to: '/FgImageContainer',
+        label: 'Fg Image Container',
```

Comment:
What do you guys think, should the side menu also have a prefix? 
Seems like frameworks like Vuetify & Element UI does not have this. @zashton @logan-laughery. 
My vote goes for not having prefix on component "titles".

I will add suggestions on the rest too, but dont accept these ones yet 😄 @zashton 
```suggestion
        label: 'Image Container',
```

Reply:
I agree.  In my opinion, not having the prefix on the side menu helps make the documentation easier for newcomers to explore.